azurerm/internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -227,7 +234,6 @@ func resourceArmCosmosDbAccount() *schema.Resource {
 								"mongoEnableDocLevelTTL",
 								"DisableRateLimitingResponses",
 								"AllowSelfServeUpgradeToMongo36",
-								"EnableAnalyticalStorage",
```

Comment:
This would be a breaking change, could we keep it in with a comment to remove in 3.0? (but we can remove it from the docs)

Reply:
@katbyte well... it won't be, in fact :), since this could have never worked. :) It's never ben a capability from what I can see. I tried reproducing the bug linked, see my comment there, it just doesn't work at all.

Reply:
Other than that, I need to rebase this one on master, since @mbfrahry merged new test shim today, so leave this with me, I'll fix it all tomorrow. If you insist on leaving this in - I will, but I'm 99% positive this was broken from inception.

Reply:
Its mostly if someone is using it (regardless of if it was working or not) we would break their tf plan/apply which is less then ideal

Reply:
No prob, I'll leave it in.